Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan.yml

```diff
@@ -1602,40 +1602,33 @@ tasks:
     task:
       id: b69262ea-4383-4218-824b-e13233a2c5a1
       version: -1
-      name: Isolate Endpoint - Generic
+      name: Isolate Endpoint - Generic V2
       description: |-
-        This playbook isolates a given endpoint using the following integrations:
-        - Carbon Black Enterprise Response
-        - Palo Alto Networks Traps
-      playbookName: Isolate Endpoint - Generic
+        This playbook isolates a given endpoint using various endpoint product integrations.
+        Make sure to provide valid playbook inputs for the integration you are using.
+      playbookName: Isolate Endpoint - Generic V2
       type: playbook
       iscommand: false
       brand: ""
     nexttasks:
       '#none#':
       - "61"
     scriptarguments:
-      EndpointId:
-        complex:
-          root: PaloAltoNetworksXDREndpoint
-          filters:
-          - - operator: in
-              left:
-                value:
-                  simple: PaloAltoNetworksXDREndpoint.endpoint_name
-                iscontext: true
-              right:
-                value:
-                  simple: AttackerHostnames
-                iscontext: true
-          accessor: endpoint_id
-          transformers:
-          - operator: uniq
-      Hostname:
+      Endpoint_hostname:
         complex:
           root: AttackerHostnames
           transformers:
           - operator: uniq
+      Endpoint_id:
+        complex:
+          root: |2-
```

Comment:
I think you have space or enter here

Reply:
Don't see an issue here... Tested both here on GitHub and on PyCharm.

Reply:
Do you have an env? Sometimes it can be also something like enter or whitespace that somehow was added there - you can see that in other PRs you don't have this |2-

Reply:
Oh you were referring to the `root: |2-` part. I thought you were talking about bad indentation on the YAML file.
I'll fix that.